www/includes/easyparliament/hansardlist.php

```diff
@@ -2757,6 +2757,126 @@ public function biggest_debates($args=array()) {
         return $this->_get_data_by_biggest_debates($args);
     }
 
+    public function _get_data_by_featured_gid($args=array()) {
```

Comment:
Could this be factored in with recent debates, it seems to be quite similar to it?